hwilib/devices/keepkey.py

```diff
@@ -169,6 +169,14 @@ def can_sign_taproot(self) -> bool:
         """
         return False
 
+    def can_sign_taproot(self) -> bool:
+        """
+        The KeepKey does not support Taproot yet.
+
+        :returns: False, always
+        """
+        return False
+
```

Comment:
`can_sign_taproot` is already defined above and is a duplicate

Reply:
Oops, fixed.